wit/wit_test.go

```diff
@@ -0,0 +1,74 @@
+//go:build !tinygo
+// +build !tinygo
+
+package wit
+
+import (
+	"bytes"
+	"context"
+	"io/fs"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/sergi/go-diff/diffmatchpatch"
+	"go.bytecodealliance.org/internal/wasmtools"
+)
+
+func TestGoldenWITRoundTrip(t *testing.T) {
+	if testing.Short() {
+		// t.Skip is not available in TinyGo, requires runtime.Goexit()
+		return
+	}
+	ctx := context.Background()
+	wasmTools, err := wasmtools.New(ctx)
```

Comment:
Can this mechanism be used to skip this test in TinyGo?

Reply:
what mechanism?

Reply:
Rather than have a build tag on tests, the test can see the error returned from wasmtools.New and skip the test.

Reply:
Ah yes, good point. I modified the `New()` function to return an error if it's run under TinyGo and reverted back the change made to the test file.